### PR TITLE
Formatted code to closer match PEP8 and removed `object` inheritance.

### PR DESCRIPTION
**Pull request by mattclarke**
Fixed some formatting where it didn't agree with PEP8.
Also, removed instances of inheriting from `object`.

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@mattclarke

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

**Comment by tirkarthi** [comment]
PEP8 only changes pollute git history and are not generally merged. There are a lot of places where object is inherited from so I am not sure of removing them all in a module is worthy enough.

**Comment by mattclarke** [comment]
I can remove them all for all modules if you want? :)

**Comment by methane** [comment]
I'm sorry, but we don't accept formatting only changes in general.

**Comment by merwok** [comment]
@mattclarke sorry your first PR was a style change!  you’re not the first one to have this idea, we know it comes from a good intention, but the CPython project doesn’t take these kind of aesthetic-only changes

- modules have been contributed by multiple people over 30 years, so there isn’t one style
- each change has a cost: reviewer time, compute time on the test machines, size of the repo and transfer, etc
- a cosmetic change doesn’t really improve or fix anything
- cosmetic changes obscure history and make it harder to find who really changed what line when (devs spend a lot of time looking at history to understand changes in a module and track where bugs come from)

I hope this explains why your PR was not accepted, and that if you want to make a contribution you are welcome!  Here’s the guide: https://devguide.python.org/

File: Lib/xml/dom/minidom.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ def insertBefore(self, newChild, refChild):
             newChild.nextSibling = refChild
             refChild.previousSibling = newChild
             if index:
-                node = self.childNodes[index-1]
+                node = self.childNodes[index - 1]
```
Comment (terryjreedy) [inline Lib/xml/dom/minidom.py:106]: Not PEP 8, makes code worse

File: Lib/xml/dom/minidom.py [hunk 2]
```diff
@@ -343,8 +349,9 @@ def __init__(self):
 
 
 class Attr(Node):
-    __slots__=('_name', '_value', 'namespaceURI',
-               '_prefix', 'childNodes', '_localName', 'ownerDocument', 'ownerElement')
+    __slots__ = ('_name', '_value', 'namespaceURI',
+                 '_prefix', 'childNodes', '_localName', 'ownerDocument',
+                 'ownerElement')
```
Comment (terryjreedy) [inline Lib/xml/dom/minidom.py:354]: Bad wrapping.

File: Lib/xml/dom/minidom.py [hunk 3]
```diff
@@ -148,7 +148,7 @@ def replaceChild(self, newChild, oldChild):
         newChild.parentNode = self
         oldChild.parentNode = None
         if (newChild.nodeType in _nodeTypes_with_children
-            or oldChild.nodeType in _nodeTypes_with_children):
+                or oldChild.nodeType in _nodeTypes_with_children):
```
Comment (terryjreedy) [inline Lib/xml/dom/minidom.py:151]: Wrong change; see section on newline before/after binary operator.
There are several other similarly wrong changes.

File: Lib/xml/dom/minidom.py [hunk 4]
```diff
@@ -657,12 +666,15 @@ def _get_name(self):
...
 
+
 class Element(Node):
-    __slots__=('ownerDocument', 'parentNode', 'tagName', 'nodeName', 'prefix',
-               'namespaceURI', '_localName', 'childNodes', '_attrs', '_attrsNS',
-               'nextSibling', 'previousSibling')
+    __slots__ = ('ownerDocument', 'parentNode', 'tagName', 'nodeName', 'prefix',
+                 'namespaceURI', '_localName', 'childNodes', '_attrs',
+                 '_attrsNS',
+                 'nextSibling', 'previousSibling')
```
Comment (terryjreedy) [inline Lib/xml/dom/minidom.py:677]: More bad wrapping.

**Review by terryjreedy — COMMENTED** [review]
There are several changes that I think are wrong.  We could discuss whether to remove the 2.x 'object' in class statements, if we have not done so before, but that should start with an issue and no PR until substantial agreement from coredevs.

**Pull request closed without merging**